### PR TITLE
adding a new e2e text and change text

### PR DESCRIPTION
**Pull request by angel2895**

### CODE REVIEW

**Comment by angel2895** [comment]
i hope that this change appears in the version of the app.

File: README.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ And we can see that all tests passed:
 
 ![test-passed-cypress](https://github.com/luisangel2895/alegra-frontend/blob/67e954f0c616d55c65062f172c3aa23ca3948a43/images/Screenshot_5.png)
 
-And finally we can see the video generated by Crypress for this integration tests.
```
Comment (luisangel2895) [inline README.md:121]: why you change the tests word?

File: README.md [hunk 2]
```diff
@@ -124,4 +124,6 @@ And finally we can see the video generated by Crypress for this integration test
 https://user-images.githubusercontent.com/16966767/155645494-7d741f23-850b-42be-b881-8599133e9d18.mp4
 
 
+Other e2e testing
 
+video-none :(
```
Comment (luisangel2895) [inline README.md]: please jus need to put the video into github directory directly, do you have any problem?

**Review by luisangel2895 — CHANGES_REQUESTED** [review]
@angel2895 please i will add the video but dont change the words of readme

**Comment by luisangel2895** [comment]
i already return the word test

**Comment by luisangel2895** [comment]
i already add a valid link to the video.

**Review by luisangel2895 — APPROVED** [review]
Perfect @angel2895  its all for today

**Comment by luisangel2895** [comment]
Perfect i'll join tomorrow

**Pull request merged**